src/udp_wrap.cc

```diff
@@ -696,27 +700,33 @@ void UDPWrap::OnRecv(ssize_t nread,
                      const sockaddr* addr,
                      unsigned int flags) {
   Environment* env = this->env();
-  AllocatedBuffer buf(env, buf_);
+  Isolate* isolate = env->isolate();
+  std::unique_ptr<BackingStore> bs = env->release_managed_buffer(buf_);
   if (nread == 0 && addr == nullptr) {
     return;
   }
 
-  HandleScope handle_scope(env->isolate());
-  Context::Scope context_scope(env->context());
+  const HandleScope handle_scope(isolate);
+  const Context::Scope context_scope(env->context());
```

Comment:
This is pretty atypical for our code – it’s not wrong, but `const` doesn’t really effect a meaningful change in semantics for scope classes

Reply:
@addaleax I did notice that we sparingly use `const`s for most of the V8 datatypes except for `FunctionCallbackInfo` but I never knew the reason why. By marking the scope object as `const`, aren't we trying to convey that nothing on that object should be changed? Isn't it like a rule of thumb to mark everything as const, unless we really need to mutate something?

Reply:
@RaisinTen For `FunctionCallbackInfo` it’s just something that the V8 API requires from function callbacks :)

It’s not bad per se, but it does look a bit odd to apply it to types that aren’t even really mutable, regardless of whether const or not.

Reply:
@addaleax got it, removed the `const`s as they seemed unnecessary